docs/usage.md

```diff
@@ -42,6 +42,7 @@ The parameters are described in details in the [online documentation](https://nf
 
 - `--m2m` enables the computation of the _many-to-many_ alignment, which is the only one to be useful in the case of self-alignments, but which on the other hand can exhaust computing resources in the case of very large genomes.
 - Likewise, when comparing very similar and repetitive genomes (like two vertebrate genomes from the same species), any dotplot other than for the _one-to-one_ alignment will be heavy to compute and useless anyway, because the whole page will be filled with dots. The `--skip_dotplot_*` options are there to solve that problem.
+- Users who need a different format than MAF can check the `--export_aln_to` parameter to generate extra files.
```

Comment:
might be good to add which format are those extra files

Reply:
As the information is in the parameters help page because the formats are enumerated in `nextflow_schema.json`, I prefer to not duplicate the information so that it does not get out of sync.